lib/file-list.js

```diff
@@ -230,6 +231,13 @@ Object.defineProperty(List.prototype, 'files', {
     var lookup = {}
     var included = {}
     this._patterns.forEach(function (p) {
+      // This needs to be here sadly, as plugins are modifiying
+      // the _patterns directly resulting in elements not being
```

Comment:
@dignifiedquire for my own edification, can you give an example of a plugin doing this? Specifically, is [karm-jspm](https://github.com/Workiva/karma-jspm) doing this?